core/errors/infer.cc

```diff
@@ -3,7 +3,7 @@
 
 namespace sorbet::core::errors::Infer {
 
-ErrorClass errorClassForUntyped(const GlobalState &gs, FileRef file) {
+ErrorClass errorClassForUntyped(const GlobalState &gs, FileRef file, SymbolRef blame) {
```

Comment:
```suggestion
ErrorClass errorClassForUntyped(const GlobalState &gs, FileRef file, const TypePtr & untyped) {
```

- This way, we don't call `untypedBlame` unless we know we're going to use it
- Using `const TypePtr &` should be faster than `TypePtr` because it won't have to increment/decrement an atomic ref count.